app/helpers/users_helper.rb

```diff
@@ -8,4 +8,13 @@ def user_avatar(user, classes)
       @avatar_user = image_tag("/avatar.jpg", alt: user.name, class: classes)
     end
   end
+
+  def admin_users(current_user,user)
```

Comment:
@ivanraybuglosa You can use `current_user` method here also, so no need to pass.